data/model/test/test_org_mirror.py

```
@@ -674,3 +681,745 @@ def test_update_org_mirror_config_preserves_unchanged_fields(self, initialized_d
+        assert result == []
+
+    def test_ready_candidates_returned(self, initialized_db):
+        """
+        Repos with sync_start_date in the past and retries remaining should be returned.
+        """
+        org, robot = _create_org_and_robot("eligible_test1")
+        config = _create_org_mirror_config(org, robot, is_enabled=True)
+
+        # Create a ready repo (past due, retries remaining, not syncing)
+        past_time = datetime.utcnow() - timedelta(hours=1)
+        repo = OrgMirrorRepository.create(
```

Ruff (0.14.13)

Local variable `repo` is assigned to but never used

Remove assignment to unused variable `repo`

(F841)